.github/workflows/publish.yml

```diff
@@ -59,6 +59,9 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.BOT_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.BOT_AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: us-east-1
+        # This is not running the integration tests of other providers. Think how this could be done, as they require some other configuration
+        # (as an example, check the k8s tests in "integration-test.yml", it requires a minicube environment)
+        # Maybe running them in different jobs
```

Comment:
This is important. @rdiaz82 not sure if you knew this -> The Kubernetes integration tests are not being run when releasing a new version